### PR TITLE
Adding arcestia.json for testing purpose

### LINKED ISSUE #1: Ini hanya sebuah test saja
Let's start exploring

### PR DESCRIPTION
**Pull request by arcestia**
ohhhhhh la... la... la... la....

### CODE REVIEW

File: arcestia.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "basics": {
+    "name": "Laurensius Jeffrey Chandra",
+    "label": "Human Right Activist. Globalist and Investor.",
+    "image": "https://avatars1.githubusercontent.com/u/7936962?s=460&u=41e34bb816ad09323e1650f3efc0bec4fb2bc5dd&v=4",
+    "summary": "I’m now just a trader, investor who still like to code at freetime.",
```
Comment (DarkGameMaster) [inline arcestia.json]: "summary": "I’m now just a trader, investor who still like to code at freetime.",

to

"summary": "I’m now just a trader and also an investor, who still like to code at free time.",

Comment (arcestia) [reply arcestia.json]: wow a mistake spelling hmm, interesting

File: arcestia.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "basics": {
+    "name": "Laurensius Jeffrey Chandra",
+    "label": "Human Right Activist. Globalist and Investor.",
+    "image": "https://avatars1.githubusercontent.com/u/7936962?s=460&u=41e34bb816ad09323e1650f3efc0bec4fb2bc5dd&v=4",
+    "summary": "I’m now just a trader and also an investor, who still like to code at free time.",
+    "website": "https://arcestia.com",
+    "email": "me@arcestia,com",
+    "email2": "vrusolik@gmail.com",
```
Comment (arcestia) [inline arcestia.json:9]: "email2": "vrusolik@gmail.com",
it should better
"email_2": "vrusolik@gmail.com",

File: arcestia.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+  "basics": {
+    "name": "Laurensius Jeffrey Chandra",
+    "label": "Human Right Activist. Globalist and Investor.",
+    "image": "https://avatars1.githubusercontent.com/u/7936962?s=460&u=41e34bb816ad09323e1650f3efc0bec4fb2bc5dd&v=4",
+    "summary": "I’m now just a trader and also an investor, who still like to code at free time.",
```
Comment (arcestia) [inline arcestia.json:6]: make it more descriptive.

File: arcestia.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (arcestia) [inline arcestia.json:1]: rename it to resume.json and re-edit with jsonresume.org/schema/.

**Review by arcestia — COMMENTED** [review]
just review test

**Pull request merged**